### PR TITLE
Update README and delete username prop in picgo.picBed.github

### PR DESCRIPTION
**Pull request by Spades-S**
Update README and delete username prop in picgo.picBed.github

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,8 +38,8 @@
   * Fix a bug caused by settings.json with comments.
 
 * 1.0.0  
-  * Change repository's url and issue's url.
-  * Upgrade PicGo-Core dependency: from ^1.1.5 to ^1.1.9.
+  * Change respotory's url and issus's url.
+  * Upgrade Picgo-Core dependiency: from ^1.1.5 to ^1.1.9.
```
Comment (upupming) [inline CHANGELOG.md]: It seems that you have reverted my changes, the typos come again.
Could you please undo these 'changes'?

File: README.md [hunk 2]
```diff
@@ -9,108 +11,71 @@
...
+`vs-picgo` is a VSCode extension for uploading images to a remote image hosting service and insert the url into current active MarkDown file. It's much more efficient than other tools. And it can give us better experience of uploading images. `vs-picgo` supports 8 kinds of image hosting services, [weibo](https://picgo.github.io/PicGo-Doc/zh/guide/config.html#%E5%BE%AE%E5%8D%9A%E5%9B%BE%E5%BA%8A), [qiniu](https://picgo.github.io/PicGo-Doc/zh/guide/config.html#%E4%B8%83%E7%89%9B%E5%9B%BE%E5%BA%8A), [tcyun](https://picgo.github.io/PicGo-Doc/zh/guide/config.html#%E8%85%BE%E8%AE%AF%E4%BA%91cos), [upyun](https://picgo.github.io/PicGo-Doc/zh/guide/config.html#%E5%8F%88%E6%8B%8D%E4%BA%91), [github](https://picgo.github.io/PicGo-Doc/zh/guide/config.html#github%E5%9B%BE%E5%BA%8A), [aliyun](https://picgo.github.io/PicGo-Doc/zh/guide/config.html#%E9%98%BF%E9%87%8C%E4%BA%91oss), [imgur](https://picgo.github.io/PicGo-Doc/zh/guide/config.html#imgur%E5%9B%BE%E5%BA%8A), [SM.MS](https://sm.ms/) ,supported by [PicGo-Core](https://github.com/PicGo/PicGo-Core).
```
Comment (upupming) [inline README.md]: should replace `[ ],` with `,[ ]`, small mistake haha
> [SM.MS](https://sm.ms/) ,supported by [PicGo-Core](https://github.com/PicGo/PicGo-Core).

File: CHANGELOG.md (hunk 1 above)
Comment (Spades-S) [reply CHANGELOG.md]: SURE! I have no mind to do these changes, I'll undo these.

File: README.md (hunk 2 above)
Comment (Spades-S) [reply README.md]: @upupming  Of great importance.

**Pull request merged**